### PR TITLE
Batchrunner

### PR DESCRIPTION
**Pull request by ivanbusthomi**
improvement for batch runner. still buggy :)

### CODE REVIEW

**Comment by Gustry** [comment]
Hi @ivanbusthomi I known you don't have the pep8 command on your machine. But Travis is running this command for you : https://travis-ci.org/inasafe/inasafe/builds/171936827#L373 ;-)
We are guilty for the exposure file I think ;-)

We can fix that later, just to let you know. You can configure pycharm to not be happy when the line is longer than 80 chars.
screen shot 2016-11-07 at 15 45 54

There is still some comments, are they useful ?

**Comment by timlinux** [comment]
@ivanbusthomi after addressing @Gustry 's comment please merge unless there is still something on your todo list. Thanks!

File: safe/gui/tools/batch/batch_dialog.py [hunk 1]
```diff
@@ -679,7 +741,7 @@ def on_output_directory_chooser_clicked(self):
     def help_toggled(self, flag):
         """Show or hide the help tab in the main stacked widget.
 
-        .. versionadded: 3.2.1
+        .. version added: 3.2.1
```
Comment (Gustry) [inline safe/gui/tools/batch/batch_dialog.py]: For next time, it was correct without a space. See http://www.sphinx-doc.org/en/1.4.8/markup/para.html#directive-versionadded

File: safe/gui/tools/batch/batch_dialog.py [hunk 2]
```diff
@@ -694,7 +756,7 @@ def help_toggled(self, flag):
     def hide_help(self):
         """Hide the usage info from the user.
 
-        .. versionadded:: 3.2.1
+        .. version added:: 3.2.1
```
Comment (Gustry) [inline safe/gui/tools/batch/batch_dialog.py]: Same as above

File: safe/gui/tools/batch/batch_dialog.py [hunk 3]
```diff
@@ -500,14 +524,52 @@ def run_task(self, task_item, status_item, count=0, index=''):
...
+
+                # print layer list. somehow this is the only way to update
+                # the map canvas. still need improvement
+                layers = self.iface.mapCanvas().layers()
+                for layer in layers:
+                    print layer.name()
                 # noinspection PyBroadException
+                # status_item.setText(self.tr('Analysis Ok'))
+                # self.create_pdf(title, path, cloned_layer, count, index)
+                # status_item.setText(self.tr('Report OK'))
```
Comment (Gustry) [inline safe/gui/tools/batch/batch_dialog.py]: Usually, we remove commented code. Do you need to let it ?

**Comment by Gustry** [comment]
Thanks a lot for the PEP8 ! It looks nice now.
If you want to merge, it's fine, or if you have time to just fix these small comments, it's better ;-)

File: safe/gui/tools/batch/batch_dialog.py (hunk 3 above)
Comment (Gustry) [inline safe/gui/tools/batch/batch_dialog.py]: Ok, thanks a lot. Then, this PR will be good ;-)

Comment (ivanbusthomi) [inline safe/gui/tools/batch/batch_dialog.py]: sorry @Gustry , I forgot pushing the changes to the branch a few days ago, and when I check it right now I just realised it.
I already pushed it now.

**Comment by Gustry** [comment]
Hi @ivanbusthomi , will you fix the sphinx docstring ? `versionadded`

**Comment by ivanbusthomi** [comment]
Hi @Gustry and @timlinux ,
I've pushed some commits. If there are some parts in the code that you think I can improve, please let me know. 
> @ivanbusthomi after addressing @Gustry 's comment please merge unless there is still something on your todo list. Thanks!

Currently, I don't have the access to merge my PR so if you think it is OK I would be glad if you can merge it. Thanks!

**Comment by Gustry** [comment]
Thanks @ivanbusthomi , I will merge it !

**Pull request merged**